src/styles/card.scss

```diff
@@ -0,0 +1,99 @@
+@import 'variables';
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: $border-radius;
+  padding: 32px 16px 16px;
+  transition-duration: $transition-duration;
+
+  &__title {
+    font-weight: 500;
+    font-size: $font-size-medium;
+    line-height: 18px;
+    margin: 0 0 4px;
+    color: $main-accent-color;
+  }
+
+  &:hover {
+    transform: scale(1.2);
```

Comment:
would be better to add `transition: transform 0.3s;` for some transition duration